manifest.json

```diff
@@ -35,7 +35,6 @@
         "extension_pages": "script-src 'self' 'wasm-unsafe-eval'; object-src 'self';",
         "sandbox": "sandbox allow-scripts allow-downloads allow-forms allow-popups allow-modals; script-src 'self' 'unsafe-inline' 'unsafe-eval'; child-src 'self';"
     },
-
```

Comment:
Could you add this whitespace back in (carriage return) here? No need for this file to be part of this PR, and it's good to keep this file an almost identical version of the other browser extensions manifests (which have this space).